dev_guide/builds.adoc

```diff
@@ -770,6 +776,9 @@ source:
 ----
 ====
 
+You can also 
+xref:../install_config/http_proxies.adoc#configuring-hosts-for-proxies[configure a proxy for git cloning using Ansible].
+
```

Comment:
@tpoitras For the changes in this topic in general, remember that `install_config` won't exist in the Online/OSD docs.